api/net/Areas/Subscriber/Controllers/AVOverviewController.cs

```diff
@@ -0,0 +1,67 @@
+using System.Net;
+using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Helpers;
+using TNO.API.Models;
+using TNO.Core.Exceptions;
+using TNO.DAL.Services;
+using TNO.Keycloak;
+
+namespace TNO.API.Areas.Subscriber.Controllers;
+
+/// <summary>
+/// AVOverviewController class, provides endpoints to manage evening overviews.
+/// </summary>
+[ClientRoleAuthorize(ClientRole.Editor)]
+[ApiController]
+[Area("subscriber")]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[area]/reports/av/overviews")]
+[Route("api/[area]/reports/av/overviews")]
+[Route("v{version:apiVersion}/[area]/reports/av/overviews")]
+[Route("[area]/reports/av/overviews")]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Unauthorized)]
+[ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.Forbidden)]
+public class AVOverviewController : ControllerBase
```

Comment:
@jdtoombs I thought the Evening Overview was visible on the Subscriber app?  If it is, I don't know how it can be currently working correctly because there was no endpoints for it.